.github/workflows/release.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 The Protobom Authors
```

Comment:
Should we update the license header

Reply:
Been caught stealin'!
![IMAGE](https://media2.giphy.com/media/DWxKllu8w3ZxmFU4zN/200w.gif?cid=6c09b952kiakhrp6n052gtkra22ajru5hmmoocst2kuv3sxt&ep=v1_gifs_search&rid=200w.gif&ct=g)